normalizer_lib.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""Library of language-agnostic FST rewrite rules to normalize text."""
+
+import unicodedata
+from pynini import *
+from config import *
+
+LANGUAGE = af
```

Comment:
language agnostic?  We should use a sample language only in the test file and keep this all language neutral.

Reply:
The only things in this file that rely on the language are loading in the graphemes/punctuation and the language-specific rewrite rules. Perhaps we can have a main config that says the language we're using and/or which data sources we want to normalize and call it with something like `LANGUAGE = main_conf.LANGUAGE`?

One other possibility is turning it into a flag in `normalizer.py`, but I don't know how that would play with `normalizer_test.py`. Although, if the test file only uses the sample language and doesn't need language-specific anything, perhaps a flag could work if we can have it trickle into `normalizer_lib` when we run the binary.